README.md

```diff
@@ -1 +1,94 @@
-# The javascript web client SDK for blindnet
+<h1 align="center">
+  blindnet devkit<br />Javascript Client SDK
+</h1>
+
+<p align=center><img src="https://user-images.githubusercontent.com/7578400/163277439-edd00509-1d1b-4565-a0d3-49057ebeb92a.png#gh-light-mode-only" height="80" /></p>
+<p align=center><img src="https://user-images.githubusercontent.com/7578400/163549893-117bbd70-b81a-47fd-8e1f-844911e48d68.png#gh-dark-mode-only" height="80" /></p>
+
+<p align="center">
+  <strong>JavaScript <a href="https://blindnet.dev/docs/extra/glossary/#client-side-sdk">client-side SDK</a> for e2e encryption and users management.</strong>
```

Comment:
Structure of the docs changed a bit. Remove `/extra` from every _blindnet.dev_ link in the README